platforms/core-runtime/build-option/src/main/java/org/gradle/internal/buildoption/DefaultFeatureFlags.java

```diff
@@ -18,27 +18,27 @@
 
 import org.gradle.internal.event.ListenerManager;
 
+import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.CopyOnWriteArraySet;
 
 public class DefaultFeatureFlags implements FeatureFlags {
```

Comment:
💭 Interesting that this thing is actually backing a public API, it should never have used `InternalOptions` to begin with.